jnosql-mongodb/pom.xml

```diff
@@ -44,6 +45,27 @@
             <artifactId>mongodb-driver-sync</artifactId>
             <version>${monbodb.driver}</version>
         </dependency>
+        <dependency>
+            <groupId>jakarta.nosql</groupId>
+            <artifactId>jakarta.nosql-tck</artifactId>
+            <version>${jakarta.nosql.version}</version>
+            <scope>test</scope>
+        </dependency>
     </dependencies>
 
+    <build>
```

Comment:
Here are my 5 cents on your idea:

To improve this utilization, let's declare this <build> section under a specific maven profile, called `tck` for example, into the `jnosql-databases-parent\pom.xml` and conditioning its execution by a property called `nosql.tck.skip` e.g. With that all the JNoSQL Database Implementation will gonna be able to use it just calling `mvn test -Ptck`. Another actions that we would need to have to do:

- Add the required `TemplateSupplier` class and its service provider declaration at `META-INF\services` on each JNoSQL Database Implementation that supports Jakarta NoSQL TCK execution;

- For JNoSQL Database implementations that do not support Jakarta NoSQL TCK we should define the `nosql.tck.skip` to `true`;